src/vl-multiselect.js

```diff
@@ -24,7 +24,15 @@ export class VlMultiSelect extends VlSelect {
 
   connectedCallback() {
     this.classList.add('vl-multiselect');
+    this.setAttribute('multiple', 'multiple');
```

Comment:
Attribuut `multiple` is een boolean attribuut, kan in dit geval geen kwaad aangezien de string ook truthy geeft, maar ik zou eerder een lege string als value hanteren.

Reply:
Fixed.